src/components/views/dialogs/CreateRoomDialog.tsx

```diff
@@ -59,19 +60,23 @@ interface IState {
 
 export default class CreateRoomDialog extends React.Component<IProps, IState> {
     private readonly supportsRestricted: boolean;
+    private readonly knockingEnabled: boolean;
     private nameField = createRef<Field>();
     private aliasField = createRef<RoomAliasField>();
 
     constructor(props) {
         super(props);
 
         this.supportsRestricted = !!this.props.parentSpace;
+        this.knockingEnabled = SettingsStore.getValue("feature_knocking");
 
         let joinRule = JoinRule.Invite;
         if (this.props.defaultPublic) {
             joinRule = JoinRule.Public;
         } else if (this.supportsRestricted) {
             joinRule = JoinRule.Restricted;
+        } else if (this.knockingEnabled) {
+            joinRule = JoinRule.Knock;
```

Comment:
The designs do not seem to suggest that we should default to knocking - see https://www.figma.com/file/7TqjqdMBaPpm3IavKsMYu6/%5BBWI%5D-Knocking?node-id=367%3A50166